utils/wesnoth-map-diff/images/.gitignore

```diff
@@ -0,0 +1 @@
+!.
```

Comment:
We don't need this folder
```suggestion
```

Reply:
it's still referenced at line 49 of `images.ts`: https://github.com/wesnoth/wesnoth/pull/7452/files#diff-e229e38d955fc255563574ebed9dc70546a7fba2852ebf7ef1fe3a901cffcc01R49

Reply:
We might still put something in it later, though

Reply:
...and there. Now, as of [0bbab66](https://github.com/wesnoth/wesnoth/pull/7452/commits/0bbab662312bb53941d1df2f0e35f487000535a9), there is something in it.